.github/workflows/qc.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ["3.7", "3.10"]
```

Comment:
Just for me to understand, why did you remove 3.8, 3.9?

Reply:
If it works on 3.7 and 3.10 it's pretty good chance it works in the middle. I also needed to make a change to get it to run again 😆 . We're still not ready for 3.10 apparently